benchmarks/Benchmarks/CertificatesBenchmarks/Benchmarks.swift

```diff
@@ -0,0 +1,35 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the SwiftCertificates open source project
+//
+// Copyright (c) 2023 Apple Inc. and the SwiftCertificates project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of SwiftCertificates project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+import Benchmark
+import Sources
+
+let benchmarks = {
+    Benchmark("Verifier", configuration: .init(warmupIterations: 1)) { benchmark in
```

Comment:
Why do we warm up here but not in the others?

Reply:
We do access a bunch of `static let`s that are initialised on first access. "Parse WebPKI Roots" should get it too but the others have not shown any benefit. But it also doesn't hurt AFAIK so I'm open to add it everywhere.

Reply:
Might be worth adding a comment for that.